api/models.py

```diff
@@ -7,6 +7,7 @@
 from django.contrib.postgres.fields import HStoreField
 from django.db import models
 from django.utils.encoding import python_2_unicode_compatible
+from django.utils import timezone
```

Comment:
You also can just import the `now` function, because you don't need/use anything else of the timezone namespace.